react/components/EXPERIMENTAL_Table/docs/Features.md

```diff
@@ -0,0 +1,561 @@
+### Measures
+
+The table measures are the current density, rowHeight, tableHeight and setCurrentDensity. They are calculated by the `useTableMeasures` hook.
+
+```ts
+// Density shape
+enum Density {
+  Compact = 'compact',
+  Regular = 'regular',
+  Comfortable = 'comfortable',
+}
+```
+
+##### useTableMeasures Inputs
+
+| Property | Type    | Required | Default         | Description               |
+| -------- | ------- | -------- | --------------- | ------------------------- |
+| size     | number  | ✅       | 0               | Length of the table items |
+| density  | Density | 🚫       | Density.Regular | Initial Density           |
+
+##### useTableMeasures Outputs
+
+| Property          | Type                       | Description               |
+| ----------------- | -------------------------- | ------------------------- |
+| currentDensity    | Density                    | The table current density |
```

Comment:
Maybe this could be renamed to just `density` before the new major... But just a thought

Reply:
yes, it definitely could. would be cleaner